install/kubernetes/helm/istio-init/files/crd-all.gen.yaml

```diff
@@ -96,6 +96,8 @@ spec:
     kind: AuthorizationPolicy
     plural: authorizationpolicies
     singular: authorizationpolicy
+    shortNames:
+    - authzpol
```

Comment:
Thanks for the PR, I'm wondering can we just make it `authz`?

Reply:
Fine with me. I choose authzpol as this resource seems a bit similar to network policies. It is a list, so we could add both?

Reply:
Yeah, I think we can do both.